Vagrantfile-windows.template

```diff
@@ -16,6 +16,7 @@ Vagrant.configure("2") do |config|
     libvirt.hyperv_feature :name => 'stimer',  :state => 'on'
     libvirt.hyperv_feature :name => 'synic',   :state => 'on'
     libvirt.hyperv_feature :name => 'vapic',   :state => 'on'
+    libvirt.hyperv_feature :name => 'vpindex',   :state => 'on'
```

Comment:
Would it be possible to align the new "vpindex" line like this:

```
    libvirt.hyperv_feature :name => 'vapic',   :state => 'on'
    libvirt.hyperv_feature :name => 'vpindex', :state => 'on'
```

Currently there are some "additional" spaces:

```
    libvirt.hyperv_feature :name => 'vapic',   :state => 'on'
    libvirt.hyperv_feature :name => 'vpindex',   :state => 'on'
```

Thank you

Reply:
Aligned the spacing and left a little more space in case any hyperv features have really long names.